spec/features/destroy_card_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+feature 'Delete Card', 'possible remove the card' do
+
+  given!(:card) { create(:card) }
+
+  scenario 'delete card' do
+    visit cards_path
+    click_on 'Удалить'
+    expect(current_path).to eq cards_path
+    expect(page).to_not have_content 'Удалить'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.